docs/rules/key-spacing.md

```diff
@@ -183,6 +184,106 @@ call({
 });
 ```
 
+### align
+
+The `align` option can take additional configuration through the `beforeColon`, `afterColon`, `mode`, and `on` options.
+
+If `align` is defined as an object, but not all of the parameters are provided, undefined parameters will default to the following:
+
+```js
+// Defaults
+align: {
+  "beforeColon": false,
+  "afterColon": true,
+  "on": "colon",
+  "mode": "strict"
+}
+```
+
+Examples of **correct** code for this rule with sample `{ "align": { } }` options:
+
+```js
+/*eslint key-spacing: ["error", {
+  "align": {
+    "beforeColon": true,
+    "afterColon": true,
+    "on": "colon"
+  }
+}]*/
+
+var obj = {
+  "one"   : 1,
+  "seven" : 7
+}
+```
+
+```js
+/*eslint key-spacing: ["error", {
+  "align": {
+    "beforeColon": false,
+    "afterColon": false,
+    "on": "value"
+  }
+}]*/
+
+var obj = {
+  "one":  1,
+  "seven":7
+}
+```
+
+### align and multiLine
+
+The `multiLine` and `align` options can differ, which allows for fine-tuned control over the `key-spacing` of your files.  `align` will **not** inherit from `multiLine` if `align` is configured as an object.
```

Comment:
I can see it illustrated in the correct example below, but it's kind of subtle and I think it would help to call it out explicitly.